packages/react-ui-core/src/IncrementDecrementFilter/IncrementDecrementFilter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class IncrementDecrementFilter extends Component {
```

Comment:
I'd like to rename this component to `Counter`.  @lfender6445 what do you think? The name currently is a little too long IMO and it's really _not_ a filter either. Yes, technically on rent-js it is, but on react-ui it should be agnostic to what it's being used for.